spec/factories/exemption.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :exemption, class: WasteExemptionsEngine::Exemption do
+    category { 0 }
+    url { "https://example.gov.uk/guidance/waste-exemptions-using-waste" }
+    summary { "Use of spam in cooking" }
+    description { "Use of spam in cooking using suitable spam rather than virgin spam or spam which has ceased to be spam - for example by cooking spam with chips." }
+    guidance { "This exemption allows you to use suitable spam rather than virgin spam or spam which has ceased to be spam - for example by cooking spam with chips." }
+
+    sequence :code do |n|
+      "F#{n}"
+    end
+  end
```

Comment:
Would be good if we find a way to load the engine factories in the projects spec. We should research such a solution :D

Reply:
I believe the old WEX code does do this so we can take a look at that.

Word of warning, it does make the dev workflow a little more messy. Say you want to add a trait for a test in the back office, you now need to open the waste-exemptions-engine, make the change, push, review, merge, then update this app to the latest to see it.

But lets have a discussion in our next dev meeting cause I'm sure there is something we can do to avoid the duplication.

Reply:
Yeah 🤔 one for discussion :D